src/main/java/org/osiam/resources/exception/OsiamExceptionHandler.java

```diff
@@ -101,45 +101,21 @@ public JsonErrorResult handleUnrecognizedProperty(Exception e) {
     @ExceptionHandler(JsonMappingException.class)
     @ResponseStatus(HttpStatus.CONFLICT)
     @ResponseBody
-    public JsonErrorResult handleJsonMapping(Exception e) {
+    public ErrorResponse handleJsonMapping(Exception e) {
         LOGGER.error("Unable to deserialize", e);
         return produceErrorResponse(e.getMessage(), HttpStatus.CONFLICT, new JsonMappingMessageTransformer());
     }
 
-    private JsonErrorResult produceErrorResponse(String message, HttpStatus status) {
+    private ErrorResponse produceErrorResponse(String message, HttpStatus status) {
         return produceErrorResponse(message, status, null);
     }
 
-    private JsonErrorResult produceErrorResponse(String message, HttpStatus status,
+    private ErrorResponse produceErrorResponse(String message, HttpStatus status,
                                                  ErrorMessageTransformer transformer) {
```

Comment:
wrong indentation, either move it 2 spaces to the left or indent the line with 8 spaces.